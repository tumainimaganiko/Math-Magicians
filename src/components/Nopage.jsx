const Nopage = () => {
  const noPage = {
    "text-align": "center",
    "font-weight": "700",
  };
  return (
    <div style={noPage}>
      <p>
        This page doesn&apos;t exist
        <br /> Please check your URL or return to Our home page
      </p>
    </div>
  );
};

export default Nopage;
