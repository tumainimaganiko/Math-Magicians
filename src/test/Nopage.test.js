import renderer from 'react-test-renderer';
import Nopage from 'components/Nopage';

it('Rendere no page', () => {
  const tree = renderer.create(<Nopage />).toJSON();
  expect(tree).toMatchSnapshot();
});
