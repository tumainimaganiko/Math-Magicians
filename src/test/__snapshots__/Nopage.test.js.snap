// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendere no page 1`] = `
<div
  style={
    Object {
      "font-weight": "700",
      "text-align": "center",
    }
  }
>
  <p>
    This page doesn't exist
    <br />
     
    Please check your URL or return to Our home page
  </p>
</div>
`;
